packages/fxa-settings/src/pages/Signup/index.test.tsx

```diff
@@ -468,26 +468,6 @@ describe('Signup page', () => {
         });
       });
 
-      it('shows error for non matching passwords', async () => {
```

Comment:
Should keep this test for sync signup flow?

Reply:
I believe we want to use the checks and Xs on the list item to indicate errors.  I will confirm in the jira.